avail-server/src/main/kotlin/com/avail/server/configuration/EnvironmentConfigurator.kt

```diff
@@ -52,10 +52,11 @@ import com.avail.utility.configuration.Configurator
  *  sourceDirectory ::= [^;]+ ;
  *  ```
  *  `logicalRoot` represents a logical root name. `objectRepository` represents
- *  the absolute path of a binary module repository. `sourceDirectory` represents
- *  the absolute path of a package, i.e., a directory containing source modules,
- *  and may be sometimes be omitted (e.g., when compilation is not required).
- *  * **AVAIL_RENAMES**: The path to the [renames file][RenamesFileParser].
+ *  the absolute path of a binary module repository. `sourceDirectory`
+ *  represents the absolute path of a package, i.e., a directory containing
+ *  source modules, and may be sometimes be omitted (e.g., when compilation is
+ *  not required). * **AVAIL_RENAMES**: The path to the
```

Comment:
Move "* **" to the a new line, and leave a blank line before it to avoid maintenance problems.